README.md

```diff
@@ -39,7 +39,7 @@ These tests use WireMock to stub the application's dependencies on the prisonApi
 ### Running the Cypress tests
 
 You need to fire up the wiremock server first:
-```docker-compose -f docker-compose-test.yaml up```
+```docker compose -f docker-compose-test.yaml up```
```

Comment:
the test compose file targets version `3.8` which is only compatible with `docker compose` as a cli plugin to docker